assets/js/index.js

```diff
@@ -0,0 +1,325 @@
+      } else {
+        document.getElementById("countdown").textContent = 0;
+      }
+    } else {
+      renderSuccessAlert();
+    }
+
+    // go to next question
+    currentQuestionIndex += 1;
+
+    // check if last question
+    if (currentQuestionIndex < kArtistsQuestions.length) {
+      // render the next question
+      removeQuestionContainer();
+      renderQuestionContainer();
+    } else {
+      if (count > 0) {
+        renderScoreForm();
+      } else {
+        removeQuestionContainer();
+        renderGameOver();
+      }
+    }
+  }
+};
+const constructQuestionContainer = function (question) {
+  // construct container div
+  const questionContainer = document.createElement("div");
+  questionContainer.setAttribute("class", "container question-container");
+  questionContainer.setAttribute("id", "question-container");
+  questionContainer.setAttribute("data-correct", question.correctOption);
+
+  // construct h2 element
+  const questionH2 = document.createElement("h2");
+  questionH2.setAttribute("class", "question");
+  questionH2.textContent = question.title;
+
+  // construct options div
+  const options = constructOptions(question.options);
+
+  // appending h2 and options div to container div
+  questionContainer.append(questionH2, options);
+
+  // add event listener to listen for click events
+  questionContainer.addEventListener("click", verifyAnswer);
+
+  return questionContainer;
+};
+
+// render question container
+const renderQuestionContainer = function () {
+  // get the current question
+  const currentQuestion = kArtistsQuestions[currentQuestionIndex];
+
+  // construct the HTML for the question container
+  const questionContainer = constructQuestionContainer(currentQuestion);
+
+  // append the container to the document
+  document.getElementById("main-container").appendChild(questionContainer);
+};
+
+const removeStartContainer = function () {
+  // target start container
+  const startContainer = document.getElementById("start-container");
+  // remove from document
+  startContainer.remove();
+};
+
+const removeQuestionContainer = function () {
+  // target question container
+  const questionContainer = document.getElementById("question-container");
+  // remove from document
+  questionContainer.remove();
+};
+
+const renderGameOver = function () {
+  const divContainer = document.createElement("div");
+  divContainer.setAttribute("class", "container game-over");
+
+  const h2Element = document.createElement("h2");
+  h2Element.textContent = "GAME OVER";
+
+  divContainer.append(h2Element);
+
+  document.getElementById("main-container").append(divContainer);
+};
+
+const startTimer = function () {
+  // declare the timer tick function
+  const timerTick = function () {
+    if (currentQuestionIndex >= kArtistsQuestions.length) {
+      clearInterval(timer);
+    } else if (count < 0) {
+      clearInterval(timer);
+      removeQuestionsContainer();
```

Comment:
This does not match the function name - this means that when your timer runs out, the questions container is not removed and the game over container is not rendered.